### PR TITLE
LH submission checklist content updates

### PR DESCRIPTION
**Pull request by lauraherring**
I've updated the copy on the audit submission checklist page to make the steps clearer for users.

### CODE REVIEW

**Comment by jperson1** [comment]
Screenshot of the changes below. This page is too big to nicely, sorry for the crop. 
image

File: backend/audit/templates/audit/submission-progress.html [hunk 1]
```diff
@@ -114,7 +114,7 @@ <h1 class="border-base-light margin-top-0">{{ auditee_name | default_if_none:''
 					<a class="usa-link text-bold"
 						href="/report_submission/federal-awards/{{ report_id }}">Upload the Federal Awards workbook</a>
 					<p class="margin-0">
-						Provide general information on the auditee and auditor, financial statements for the auditee entity, and information on the entity's federal program compliance.
+						This workbook should be completed by the auditee. For each federal award received, you'll need the financial and agency details. This is also were you list the number of audit findings.
```
Comment (jperson1) [inline backend/audit/templates/audit/submission-progress.html:117]: `were` --> `where`?

File: backend/audit/templates/audit/submission-progress.html [hunk 2]
```diff
@@ -207,7 +222,10 @@ <h1 class="border-base-light margin-top-0">{{ auditee_name | default_if_none:''
 				<div class="usa-icon-list__content {% if secondary_auditors_workbook.created is True %} text-green {% else %} text-base-dark {% endif %}">
 					<a class="usa-link text-bold"
 						href="/report_submission/secondary-auditors/{{ report_id | default_if_none:'' }}">Upload the Secondary Auditors workbook</a>
-				</div>
+						<p class="margin-0">
+							This workbook is only necessary if multiple auditors did the audit work. In these cases, the secondary auditors should enter their  information in this workbook.
```
Comment (jperson1) [inline backend/audit/templates/audit/submission-progress.html:226]: `their  information`
Double space! They display as one space anyway, so it's not impactful.

File: backend/audit/templates/audit/submission-progress.html [hunk 3]
```diff
@@ -191,7 +203,10 @@ <h1 class="border-base-light margin-top-0">{{ auditee_name | default_if_none:''
 				<div class="usa-icon-list__content {% if additional_UEIs_workbook.created is True %} text-green {% else %} text-base-dark {% endif %}">
 					<a class="usa-link text-bold"
 						href="/report_submission/additional-ueis/{{ report_id | default_if_none:'' }}">Upload the Additional UEIs workbook</a>
-				</div>
+						<p class="margin-0">
+							This workbook is only necessary if multiple the audit report covers UEIs. It is completed by the auditee. List the additional UEIs covered by the audit, excluding the primary UEI.
```
Comment (jperson1) [inline backend/audit/templates/audit/submission-progress.html:207]: `only necessary if multiple the audit report covers UEIs.`
Not sure if this needs correction, but wanted to flag it.

**Review by jperson1 — CHANGES_REQUESTED** [review]

File: backend/audit/templates/audit/submission-progress.html (hunk 1 above)
Comment (lauraherring) [reply backend/audit/templates/audit/submission-progress.html]: Good catch, thanks @jperson1! I've fixed that typo.

File: backend/audit/templates/audit/submission-progress.html (hunk 2 above)
Comment (lauraherring) [reply backend/audit/templates/audit/submission-progress.html:226]: Fixed, thanks!

File: backend/audit/templates/audit/submission-progress.html (hunk 3 above)
Comment (lauraherring) [reply backend/audit/templates/audit/submission-progress.html]: Fixed

**Comment by lauraherring** [comment]
@jperson1 I've fixed the errors you flagged. Do I need to do anything else to resubmit for a PR?

**Comment by jperson1** [comment]
@lauraherring You should be good to go! Thanks for the quick turnaround. The tests are failing due to some unrelated migrations problem that should be resolved shortly. Once it's fixed, we'll be able to merge this against main so that they pass and then merge this in.

**Review by jperson1 — APPROVED** [review]
Looks good!

**Pull request merged**